WPCliCS/ruleset.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="WPCliCS" xsi:noNamespaceSchemaLocation="../vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+	<description>Coding standard for WP-CLI projects</description>
+
+	<!--
+	To include this ruleset in a WP CLI project, use rule ref="WPCliCS" in brackets.
+
+	For help in using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage
+	For help in understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	-->
+
+	<!--
+	#############################################################################
+	USE THE PHPCompatibility RULESET
```

Comment:
Mixed case looks off.
```suggestion
	USE THE PHP COMPATIBILITY RULESET
```

Reply:
I left it as one word as it is the name of the standard, but have fully capitalized it.